src/engraving/rendering/dev/tlayout.cpp

```diff
@@ -4418,12 +4418,17 @@ void TLayout::layout(StemSlash* item, LayoutContext& ctx)
         endY = startY + up * length * sin(angle);
     }
 
-    item->setLine(LineF(PointF(startX, startY), PointF(endX, endY)));
-    item->setStemWidth(ctx.conf().styleMM(Sid::stemSlashThickness) * graceNoteMag);
+    ldata->line = LineF(PointF(startX, startY), PointF(endX, endY));
+    ldata->stemWidth = ctx.conf().styleMM(Sid::stemSlashThickness) * graceNoteMag;
 
     RectF bbox = RectF(item->line().p1(), item->line().p2()).normalized();
     bbox = bbox.adjusted(-item->stemWidth() / 2, -item->stemWidth() / 2, item->stemWidth(), item->stemWidth());
```

Comment:
Wouldn't it be good to replace `item->line()` with `ldata->line` and `item->stemWidth()` with `ldata->stemWidth`? Or are you planning to do that later in some cleanup round?

Reply:
Thank you, I replace what I notice, but I don't pay close attention, because next round - will be removing old interface, compiler show for me all places like this.